test/test_synthetic.py

```diff
@@ -0,0 +1,407 @@
+    da = ds['wet']
+    ds.close()
+    del ds
+
+    # now build the rays at the unbuffered wm nodes
+    targ_xyz = [da.x.data, da.y.data, da.z.data]
+    ray_length = length_of_ray(targ_xyz, SAobj.wmObj._zlevels, SAobj.los)
+
+    # scale by constant (units K/Pa) to match raider (m K / Pa)
+    ray_data  = ray_length * SAobj.wmObj._k2
+
+    # actual raider data
+    # undo scaling of ppm;  units are  meters * K/Pa
+    raid_data = da.data * 1e6
+
+    assert np.all(np.abs(ray_data) > 1)
+    assert np.all(np.abs(raid_data) > 1)
+
+    # normalize with the theoretical data and compare difference with 0
+    resid     = (ray_data - raid_data) / ray_data
+    np.testing.assert_almost_equal(0, resid, decimal=7)
+
+    da.close()
+    del da
+    return
+
+
+def test_wet_eq2(reg='Fort', mod='GMAO'):
+    """ Test the wet eqn (k2 part) by setting t=e and comparing to ray
+
+    The ray length is computed here (length_of_ray; m), and in raider (m * K^2/Pa)
+    ray length computed here is then scaled by constant K^2/Pa
+    ray length from raider is unscaled from parts per million
+    We check they are both large enough for meaningful numerical comparison (>1)
+    We then compute residual and normalize it we theoretical ray length
+    We ensure that normalized residual is not significantly different from 0
+        significantly different = 7 decimal places
+    """
+    ## setup the config files
+    SAobj = StudyArea(reg, mod)
+    dct_cfg      = SAobj.make_config_dict()
+    wm_dir_synth = op.dirname(SAobj.path_wm_real).replace('real', 'synth')
+    dct_cfg['runtime_group']['weather_model_directory'] = wm_dir_synth
+    dct_cfg['download_only'] = False
+
+    ## update the weather model; t = e for wet1
+    path_synth = update_model(SAobj.path_wm_real, 'wet2', wm_dir_synth)
+
+    cfg = update_yaml(dct_cfg)
+
+    ## run raider with the synthetic model
+    cmd  = f'raider.py {cfg}'
+    proc = subprocess.run(cmd.split(), stdout=subprocess.PIPE, universal_newlines=True)
+    assert np.isclose(proc.returncode, 0)
+
+    # get the just created synthetic delays
+    ds = xr.open_dataset(
+        f'{SAobj.wd}/{SAobj.WM}_tropo_{SAobj.dts.replace("_", "")}_ray.nc')
+    da = ds['wet']
+    ds.close()
+    del ds
+
+    # now build the rays at the unbuffered wm nodes
+    targ_xyz = [da.x.data, da.y.data, da.z.data]
+    ray_length = length_of_ray(targ_xyz, SAobj.wmObj._zlevels, SAobj.los)
+    # scale by constant (units K/Pa) to match raider (m K^2 / Pa)
+    ray_data  = ray_length * SAobj.wmObj._k3
+
+    # actual raider data
+    # undo scaling of ppm;  units are  meters * K^2 /Pa
+    raid_data = da.data * 1e6
+
+    assert np.all(np.abs(ray_data) > 1)
+    assert np.all(np.abs(raid_data) > 1)
+
+    # normalize with the theoretical data and compare difference with 0
+    resid     = (ray_data - raid_data) / ray_data
+    np.testing.assert_almost_equal(0, resid, decimal=7)
+
+    da.close()
```

Comment:
You can close a dataset and still have an array from that dataset open? Didn't know that.